packages/frontend-v2/package.json

```diff
@@ -14,11 +14,14 @@
     "@emotion/styled": "^11",
     "@fontsource/montserrat-alternates": "^4.5.9",
     "@graduate/common": "workspace:*",
+    "@hookform/resolvers": "^2.9.3",
+    "class-validator": "^0.13.2",
```

Comment:
why is class-validator needed on the frontend? validation should (probably) be backend-only, unless it's just field inputs (in which case we use something else, idrmb what it is)

Reply:
Cuz I was having errors with importing the dto-types into the frontend, I made a mock dto-types.ts file in the frontend temp folder. After we fix the importing dto-types error, we can remove this.

Reply:
just so you don't forget anything, can you leave a list of TODOs somewhere?